who.go

```diff
@@ -19,8 +30,10 @@ func WeAre(team string) {
   t@:
   11.
 
-
+   
    https://peak.com/jobs
    
 `)
 }
+
+who.WeAre("Devri Robotik")
```

Comment:
Bir cok insan buradaki string parametresine takilmis, benim aklima soyle bisey geldi: bu parametre, icinde `person` (people aslinda; array of person**s**) struct'inda kullandigimiz bilgileri barindiran bir JSON string olsa ve biz bundan `team`'i yaratsak nasil olur?

Reply:
Lütfen yapıp beni de ekler misiniz? Yayınıma bakabilirsiniz. Teşekkürler.